docs/docusaurus.config.js

```diff
@@ -80,10 +80,6 @@ const config = {
                 label: 'Basics',
                 to: '/docs/category/basics-',
               },
-              {
```

Comment:
Why did we remove this link? I believe we wanted to have a link in the docs section of the footer

Reply:
> Why did we remove this link? I believe we wanted to have a link in the docs section of the footer

Honestly, I thought it was a mistake since it's in both columns
![Screen Shot 2022-06-26 at 2 29 02 PM](https://user-images.githubusercontent.com/22944438/175832666-20fd1159-058c-42e5-92ca-f745c2fe4e06.png)

But if this was intentional then I'll switch it and just fix the broken link :)

Reply:
Ah I misunderstood! Nvm you’re totally right, thanks for the fix! 🚀